src/client/initialize.ts

```diff
@@ -190,6 +190,9 @@ export function initialize<T = {}>(): PluginInstance<T> {
           void execPromise('wb:plugin:element:unsubscribe:data', id);
         };
       },
+      fetchMoreElementData(id) {
```

Comment:
Type this

Reply:
All of these functions are untyped at the moment. I think I'm going to go through in a follow-up PR and add typing, rename these to `configId` and add a runtime check for `undefined` on these for better error messaging.